.github/workflows/build_and_deploy.yml

```
@@ -99,22 +122,35 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v1
+    - id: files
+      uses: jitterbit/get-changed-files@v1
+    - name: set_actions
+      if: |
+        contains(  steps.files.outputs.all, ' actions/' ) 
+        || contains(  steps.files.outputs.all, ' Dockerfile' )
+      run: echo "::set-env name=ACTIONS::true"
```

```suggestion
      run: echo "::set-env name=ACTIONS_CHANGED::true"
```
bc this is open source, might help to make them more descriptive (same with env.TRAINING)